oak_crypto/proto/v1/crypto.proto

```diff
@@ -16,6 +16,8 @@
 
 syntax = "proto3";
 
+// TODO(#4392): Currently in the process of moving to `/proto` directory.
+// DO NOT MODIFY THOSE FILES WHILE THEY ARE BEING MOVED.
 package oak.crypto.v1;
 
 option java_multiple_files = true;
```

Comment:
Can we also use "option deprecated = true;" in GitHub? If yes, maybe add it to prevent an unlikely event of someone adding a new dependency while you're getting rid of the old ones :)

Reply:
Will it prevent importing internally, since internal checks will fail (since existing rules will depend on the deprecated PR)?